### DISCUSSION TITLE
I have errors when I publish android app.

### DISCUSSION THREAD

**Discussion by kanisuki** [airsdk/Adobe-Runtime-Support#3466] [Q&A]
Please advise me.
I have two errors when I publish android app.
The error message is here.
![r60914error](https://github.com/user-attachments/assets/541975f7-72bc-4c7d-857e-7fd82f2f07da)

**Comment by ajwfrost** [comment]
Hi

The error shows it's occurring in the "resource linking" stage, but it's failing because of a filename that looks like it's a .swf file. Our SWFs should normally be in the 'assets' folder and not a resource.. are you able to confirm whether you have any files that might be packaged as resources that have non-latin filenames? Or anything in your application descriptor's manifestAdditions section that might reference a similar filename?

It's probably safer to try to keep all filenames to conform to the Java identifier restrictions i.e. starting with a-z and then only using a-z, 0-9, dot and underscore. For display names this can be anything though - so in your app descriptor, the "name" field can be whatever you want it to be (and localised), but the "filename" field should then also be provided and sticking to the latin characters.

thanks

**Reply by kanisuki** [reply]
Then I will write my google app filename in Latin characters.

thanks.
